components/alert/alert-toast.js

```diff
@@ -0,0 +1,183 @@
+import '../button/button-icon.js';
+import '../button/button-subtle.js';
+import '../colors/colors.js';
+import './alert.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+const reduceMotion = matchMedia('(prefers-reduced-motion: reduce)').matches;
+
+const states = {
+	CLOSED: 'closed', // the toast is closed
+	CLOSING: 'closing', // the close animation is running
+	PREOPENING: 'preopening', // a pause before running the opening animation because transitions won't run when changing from 'diplay: none' to 'display: block'
+	OPENING: 'opening', // the opening animation is running
+	OPEN: 'open' // the toast is open
+};
+
+class AlertToast extends LitElement {
+
+	static get properties() {
+		return {
+			buttonText: { type: String, attribute: 'button-text' },
+			hideCloseButton: { type: Boolean, attribute: 'hide-close-button' },
+			noAutoClose: { type: Boolean, attribute: 'no-auto-close' },
+			open: { type: Boolean, reflect: true },
+			subtext: { type: String },
+			type: { type: String, reflect: true },
+			_state: { type: String, reflect: true }
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+			}
```

Comment:
Normally I'd say we should include the `:host([hidden]) { display: none; }` styles here since we're overriding the default display... but it's hidden by default and the hidden-ness is controlled by the `open` attribute instead. So maybe it's fine to exclude it.